readme.md

```diff
@@ -120,6 +120,13 @@ Force enable/disable the spinner. If not specified, the spinner will be enabled
 
 Note that `{isEnabled: false}` doesn't mean it won't output anything. It just means it won't output the spinner, colors, and other ansi escape codes. It will still log text.
 
+##### consumeStdin
+
+Type: `boolean`
+Default: `false`
+
+Consume stdin input(except Ctrl+C) while running if it is TTY.
```

Comment:
It's not clear to users why they would want to turn this on. Can you include the use-case from the issue?

Reply:
done

Reply:
Well, I mean I included detailed description of what it does.